client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -118,7 +124,18 @@ export default class ReserveSubmitButton extends Vue {
 }
 </script>
 
-<style scoped lang="sass">
-.margin-top-auto
-  margin-top: auto !important
+<style scoped lang="scss">
+.margin-top-auto {
+  margin-top: auto !important;
+}
+.v-btn {
+  min-width: 125px !important;
+}
+.reserve-submit-btn {
+  color: #1669bb !important;
```

Comment:
Can we use a color variable for this (I'm assuming it is/will be used elsewhere).

Reply:
Definitely, will look if we have anything for this.

Reply:
Fiixed

Reply:
I think `$link` is also the colour we use for primary buttons... I'm thinking we should give this colour a more generic name. 

(In my commit I'm calling it "primary", hah.)

Reply:
As discussed we can leave this for now? (the name)